src/lightning_lite/utilities/types.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
```

Comment:
Moved what was needed. PL-specific type definitions remain in PL. Hence no clean file move. 
No implementation changes here.